### PR TITLE
Fixed license header to 2020

### PR DESCRIPTION
**Pull request by mteodori**
Part of Activiti/Activiti#3154

### CODE REVIEW

File: activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/assertions/ProcessAssertions.java [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2017-2020 Alfresco.
```
Comment (zoltanpalfi) [inline activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/assertions/ProcessAssertions.java]: in Activiti we have `Alfresco Software, Ltd.`, shouldn't it be the same here?

**Review by zoltanpalfi — COMMENTED** [review]
one question, but that is replicated in the whole PR

File: activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/assertions/ProcessAssertions.java (hunk 1 above)
Comment (mteodori) [reply activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/assertions/ProcessAssertions.java]: yes it should, let me check where that comes from

**Review by zoltanpalfi — APPROVED** [review]
Lgtm! 👍

**Review by zoltanpalfi — APPROVED** [review]

**Pull request merged**